cloud/amazon/s3.py

```diff
@@ -296,6 +296,15 @@ def is_walrus(s3_url):
     else:
         return False
 
+def get_md5_digest(local_file):
+    md5 = hashlib.md5()
+    with open(local_file, 'rb') as f:
```

Comment:
anything that extends the aws document fragment has a documented requirements on python >= 2.6.  Reason is that boto requires python2.6 and above.